charts/jenkins/README.md

```diff
@@ -478,6 +478,16 @@ master:
       enabled: true       # was false
 ```
 
+## Prometheus Metrics
```

Comment:
This is in the wrong section, it's in the migration guide which is why you're getting the lint error complaining about the heading level